docs/ref/templates/builtins.txt

```diff
@@ -38,6 +29,42 @@ Sample usage:
         {{ body }}
     {% endautoescape %}
 
+When auto-escaping is in effect, all content derived from variables have HTML
+escaping applied to it before placing the result into the output (but after any
+filters have been applied). This is equivalent to manually applying the
+:tfilter:`escape` filter to each variable.
+
+The only exceptions are variables that are already marked as "safe" from
+escaping: either by the code that populated the variable, or because it has had
+the :tfilter:`safe` or :tfilter:`escape` filters applied, or because it's the
+result of a previous filter that marked the string as "safe".
```

Comment:
Either should be a choice from two options. There's three here. Maybe:

```suggestion
escaping. Variables could be marked as "safe" by the code which populated the variable, the use of :tfilter:`safe` or :tfilter:`escape` filters, or because it's the result of a previous filter that marked the string as "safe".
```

Reply:
I used David's suggestion with a minor tweak.